src/models/config.ts

```diff
@@ -1,11 +1,20 @@
+export interface EmbedProps {
+  hideHeader?: boolean;
+  hideFooter?: boolean;
+};
 export interface VoyagerConfig {
   showDataSourceSelector?: boolean;
   serverUrl?: string | null;
   manualSpecificationOnly?: boolean;
+  embedProps?: EmbedProps;
```

Comment:
Let's not include unnecessary nesting.  Most of props are for using Voyager as a library anyway.  

Just make `hideHeader/Footer` properties of VoyagerConfig.